libs/entitlement/bridge/src/main/java/org/elasticsearch/entitlement/bridge/EntitlementChecker.java

```diff
@@ -29,4 +30,20 @@ public interface EntitlementChecker {
     void check$java_net_URLClassLoader$(Class<?> callerClass, String name, URL[] urls, ClassLoader parent);
 
     void check$java_net_URLClassLoader$(Class<?> callerClass, String name, URL[] urls, ClassLoader parent, URLStreamHandlerFactory factory);
+
+    // Process creation
+    void check$java_lang_ProcessHandle$of(Class<?> callerClass, long pid);
```

Comment:
why do we need to protect getting the process handle or finding its children? I thought the point was to check actual process execution.

Reply:
Not sure; it was on the list, presumably because the existing methods have SM checks. If we don't need that, I can remove those methods.

Reply:
Let's stick with just process execution for now. We can discuss separately whether we need to protect getting processing information